charts/catalog/README.md

```diff
@@ -54,6 +54,7 @@ chart and their default values.
 | `apiserver.storage.etcd.servers` | If storage type is `etcd`: etcd URL(s); override this if NOT using embedded etcd | `http://localhost:2379` |
 | `apiserver.storage.tpr.globalNamespace` | If storage type is `tpr`: Some service catalog resources are not namespaced, but third party resources must be; setting this designates a namespace that will be treated as a container for such resources | `servicecatalog` |
 | `apiserver.verbosity` | Log level; valid values are in the range 0 - 10 | `10` |
+| `apiserver.enableAuth` | Enable authentication and authorization for the service catalog API server | false |
```

Comment:
You can... but in case it's not obvious, hacking this chart's (or any chart's) included `values.yaml` isn't the _right_ way to override values. (But I'm not saying I never do it. 😉) The _right_ way (apart from using `--set`) is to provide your _own_ YAML file with overrides and point to it using `--values`. I don't think it's 100% necessary to document that here (i.e. no action required imo), but if we _do_ want to make any mention of file-driven configuration of a chart, it shouldn't encourage updating the existing `values.yaml`. One obvious danger or that is contributors who edit it can easily, accidentally submit local changes they did not intend to along with their next PR.